standard_monoids.lua

```diff
@@ -63,7 +63,6 @@ player_monoids.jump = monoid({
 	end,
 })
 
-
```

Comment:
I agree that these newlines are unnecessary, but could you put style fixes in a separate commit? It's ok if it's in the same PR as the `mod.conf` changes.